azure-pipeline-weekly.yml

```
@@ -4,8 +4,8 @@ appendCommitMessageToRunName: false
 trigger: none
 
 schedules:
-- cron: 0 6 * * 1
-  displayName: Weekly Monday build
+- cron: 0 6 * * 1,0 18 * * 4
+  displayName: Bi-Weekly Monday and Thursday build
```

I think this is read as "every 2 weeks on Monday and Thursday".
Bi-weekly can mean both.
So since we have `Monday and Thursday` there already, that's quite self-explanatory.

```suggestion
  displayName: Every Monday and Thursday morning
```

---

it's actually Thursday evening but agree

---

or should it be thursday morning, i wanted it to be exactly at the half

---

It should be morning so that we have to fix it and it's not Friday

---

updated